src/sql/workbench/services/tableDesigner/browser/tableDesignerService.ts

```diff
@@ -43,13 +43,13 @@ export class TableDesignerService implements ITableDesignerService {
 		throw invalidProvider(providerId);
 	}
 
-	public async openTableDesigner(providerId: string, tableInfo: azdata.designers.TableInfo): Promise<void> {
+	public async openTableDesigner(providerId: string, tableInfo: azdata.designers.TableInfo, telemetryInfo?: { [key: string]: string }): Promise<void> {
 		this._adsTelemetryService.createActionEvent(TelemetryView.TableDesigner, TelemetryAction.Open).withAdditionalProperties({
 			provider: providerId,
 			newTable: tableInfo.isNewTable
 		}).send();
 		const provider = this.getProvider(providerId);
-		const tableDesignerInput = this._instantiationService.createInstance(TableDesignerInput, provider, tableInfo);
+		const tableDesignerInput = this._instantiationService.createInstance(TableDesignerInput, provider, tableInfo, propertyBag);
```

Comment:
this is not going to work, right? the propertyBag variable is not even defined here.